### PR TITLE
Improves "No Appts Available"

### PR DESCRIPTION
**Pull request by harcod**
image

### CODE REVIEW

File: src/CovidAppointmentTable.js [hunk 1]
```diff
@@ -151,7 +153,21 @@ export default function CovidAppointmentTable() {
...
+                        <Alert severity={"info"}>
+                            <AlertTitle >
+                                No Appointments Found
+                            </AlertTitle>
+                                <p>
+                                    This website scrapes data from other websites every few minutes and tells you
+                                    what places currently are advertising available appointments for COVID vaccines.
+                                </p>
+                                <p>
+                                    Mass vaccination locations have the most appointments and update their schedules every Thursday.
```
Comment (johnhawkinson) [inline src/CovidAppointmentTable.js:166]: As noted in the Issue (#72), this claim about Thursdays appears to be false and unadvisible.
The first half of the sentence seems ok tho?

Comment (harcod) [reply src/CovidAppointmentTable.js:166]: Sorry.  I accidentally made the "pull request" by hitting some random key on my keyboard.  I'm going through the comments in the issue.  Thanks!

**Comment by harcod** [comment]
**_UI when there are no appointments:_**

image

**Review by j256 — APPROVED** [review]

File: src/CovidAppointmentTable.js [hunk 2]
```diff
@@ -15,6 +16,7 @@ import SignUpLink, { hasSignUpLink } from "./components/SignUpLink";
 import StaleDataIndicator from "./components/StaleDataIndicator";
 import Switch from "@material-ui/core/Switch";
 import Typography from "@material-ui/core/Typography";
+import { AlertTitle } from "@material-ui/lab";
```
Comment (livgust) [inline src/CovidAppointmentTable.js:19]: instead, please do
```
import AlertTitle from '@material-ui/lab/AlertTitle';
```
doing that kind of import saves space when things get bundled for production (IIRC)
(and then place alphabetically)

File: src/CovidAppointmentTable.js [hunk 3]
```diff
@@ -151,7 +153,28 @@ export default function CovidAppointmentTable() {
...
+                        <br/>
+                        <Alert severity={"info"}>
+                            <AlertTitle >
+                                No Appointments Found
+                            </AlertTitle>
+                                <p>
+                                    None of the vaccine sites that we monitor currently have available appointments. This website gathers data every minute from COVID-19 vaccine sites across Massachusetts.
+                                </p>
+                                <p>
+                                    Check back occasionally for updated information.
```
Comment (livgust) [inline src/CovidAppointmentTable.js:165]: Hal requested to remove the word "occasionally" here.

File: src/CovidAppointmentTable.js [hunk 4]
```diff
@@ -151,7 +153,28 @@ export default function CovidAppointmentTable() {
...
                 {ready && formattedData.length === 0 && (
                     <div role="status">
-                        <p>No appointments found.</p>
+                        <br/>
+                        <Alert severity={"info"}>
+                            <AlertTitle >
+                                No Appointments Found
+                            </AlertTitle>
+                                <p>
+                                    None of the vaccine sites that we monitor currently have available appointments. This website gathers data every minute from COVID-19 vaccine sites across Massachusetts.
```
Comment (livgust) [inline src/CovidAppointmentTable.js:162]: I'm surprised this line didn't wrap. Are you using `prettier`? If not, please make sure to run it before your code is final. Thanks!

**Review by livgust — CHANGES_REQUESTED** [review]
Hey! I know this is still marked as a draft but since we're close to a final product I wanted to jump in with comments. LMK if you have any Qs, and also please verify that all tests are passing (I think one might check for this status so it's possible it needs to be updated)

Thanks!

File: src/CovidAppointmentTable.js (hunk 4 above)
Comment (harcod) [reply src/CovidAppointmentTable.js:162]: I set my IDE option to "Prettier on save".

**Comment by harcod** [comment]
image

**Comment by j256** [comment]
This seems to be failing tests right?  Maybe:   TestingLibraryElementError: Found multiple elements with the role "alert"

**Review by livgust — APPROVED** [review]
Fixed up some unrelated warnings, quickly remedied failing unit test
Verified after fixes that all tests are passing, UI warning appears when there are no results, and copy matches what was decided on in the Conversation of this PR.

**Pull request merged**